cpp/test/DataStorm/api/Writer.cpp

```diff
@@ -230,13 +230,9 @@ void ::Writer::run(int argc, char* argv[])
             reader.waitForNoWriters();
             [[maybe_unused]] auto _ = reader.getConnectedWriters();
             [[maybe_unused]] auto connectedKeys = reader.getConnectedKeys();
-            reader.getAllUnread();
+            [[maybe_unused]] auto allUnread = reader.getAllUnread();
             reader.waitForUnread(0);
             [[maybe_unused]] bool hasUnread = reader.hasUnread();
-            if (false)
```

Comment:
I don't see the point of this `if (false)`, this API test is kind of weir with lots of unused params.

Reply:
I am surprised the compiler doesn't complain.